### PR TITLE
update de.json and add DE translation for !info

### PR DESCRIPTION
**Pull request by ReuschelCGN**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: config/locale/de.json [hunk 1]
```diff
@@ -226,7 +235,10 @@
     "Fire":"Feuer",
     "Flying":"Flug",
     "fog":"Nebel",
-    "form":"form",
+    "Forecast":"Vorhersage",
+    "form":"Form",
+    "formalola":"alolaform",
```
Comment (jfberry) [inline config/locale/de.json:240]: I think this may break tracking

File: config/locale/de.json [hunk 2]
```diff
@@ -423,6 +450,7 @@
     "Silver Pinap Berry":"Silberne Sananabeere",
     "Sinnoh Stone":"Sinnoh-Stein",
     "Sky":"Zenit",
+    "slot_changes":"Platzänderung",
```
Comment (jfberry) [inline config/locale/de.json:453]: this will need to be without _

Comment (ReuschelCGN) [reply config/locale/de.json]: yeah your right -> fixed!

File: config/locale/de.json (hunk 1 above)
Comment (jfberry) [reply config/locale/de.json:240]: ie formalola --> the regex expects form to be the parameter name, and then looks up the form type afterwards. 
ALthough maybe this means you can type alolaform and have it translated back into formalola which would work...

It makes my head hurt ;-)

File: config/locale/de.json (hunk 2 above)
Comment (ReuschelCGN) [reply config/locale/de.json]: ok i removed it! now it should be fine

File: config/locale/de.json (hunk 1 above)
Comment (ReuschelCGN) [reply config/locale/de.json]: ok i removed it! now it should be fine

**Pull request merged**